### DISCUSSION TITLE
V2 module import with scaffold option

### DISCUSSION THREAD

**Discussion by wrighteap** [drivendataorg/cookiecutter-data-science#369] [Q&A] [answered]
Hi!

New cookiecutter user here. I just tried using your new v2 release.

I created a project project, repo repo and module module, using the code scaffold option. I selected conda as my environment manager, and requirements.txt for my dependency file. 

pip list shows the package module installed at /path/to/repo. When I try to import module, I get a ModuleNotFoundError: No module named 'config', due to the import config under module/__init__.py -- Is this a design choice to invite me to add /path/to/repo/module to sys path or to modify the config directly?

Sorry in advance if the question is inane or lacking in detail - happy to be told so or asked for additional information.

**Comment by pjbull** [comment, marked as answer]
This is a real bug. Filing #370 to fix it and closing the discussion. Thanks for bringing it up @wrighteap!
